internal/pkg/skuba/addons/cilium.go

```diff
@@ -336,19 +492,19 @@ spec:
     type: RollingUpdate
   template:
     metadata:
+      annotations:
```

Comment:
is this `annotations` being empty done in purpose?

Reply:
The thing is that YAML manifest is generated from a Helm chart in Cilium repo, where this fragment looks like:

https://github.com/cilium/cilium/blob/master/install/kubernetes/cilium/charts/operator/templates/deployment.yaml#L22-L26

I can remove it from there for now and let's add Prometheus later when we have it.